Microsoft.Toolkit.Uwp/IncrementalLoadingCollection/IncrementalLoadingCollection.cs

```diff
@@ -196,31 +196,41 @@ public IAsyncOperation<LoadMoreItemsResult> LoadMoreItemsAsync(uint count)
             => LoadMoreItemsAsync(count, new CancellationToken(false)).AsAsyncOperation();
 
         /// <summary>
-        /// Clears the collection and reloads data from the source
+        /// Clears the collection and triggers/forces a reload of the first page
         /// </summary>
         /// <returns>This method does not return a result</returns>
-        [Obsolete("RefreshAsync is deprecated, please use Refresh instead.")]
-        public async Task RefreshAsync()
-        {
-            await Task.Run(() => Refresh());
-        }
-
-        /// <summary>
-        /// Clears the collection and resets the page index
-        /// which triggers an automatic reload of the first page
-        /// </summary>
-        public void Refresh()
+        public Task RefreshAsync()
```

Comment:
When was this made obsolete?

Reply:
You mean the RefreshAsync method?

Reply:
Because the fix requires that, in a particular case, the LoadMoreItemsAsync function is called. Being an asynchronous function, I think it'd be better to encourage the developers to move towards a fully asynchronous implementation.

Reply:
Since this is 3.0, shouldn't we just pull the refresh method and mark it as a breaking change for the update?

Reply:
Sure, I'll remove it

Reply:
Just removed it, also updated the sample app to use the RefreshAsync method and the PR description to reflect the breaking change.